tenseal/binding.cpp

```diff
@@ -27,6 +28,8 @@ void bind_plain_tensor(py::module &m, const std::string &name) {
         .def("shape", &type::shape)
         .def("strides", &type::strides)
         .def("size", &type::size)
+        .def("batch", &type::batch)
```

Comment:
Can we please have some tests for this feature?

Reply:
There are already c++ tests https://github.com/OpenMined/TenSEAL/blob/ckks_tensor_improvements/tests/cpp/tensors/plaintensor_test.cpp#L124

I will add python tests as well.